addons/web/static/src/scss/bootstrap_review_backend.scss

```diff
@@ -171,6 +171,11 @@ $-o-theme-text-colors: null;
     @extend %-bg-black;
 }
 
+.bg-transparent{
+    --background-color: rgba(0 0 0, var(--bg-opacity, 0));
+    background-color: var(--background-color) !important;
+}
```

Comment:
what about 
```scss
.bg-transparent {
    --bg-opacity: 0;
    @include o-print-color($black, background-color, bg-opacity);
}
```

Reply:
It's working but it would render like that :

![image](https://github.com/odoo/odoo/assets/128030743/0a8a7531-18d7-499c-9426-613ff6bd1bc2)

Reply:
I might have missed a subtilty in this, but... why not use `transparent` instead ? :thinking:

Reply:
Hi @pparidans ! I believe we tend to use the same syntax for each `bg-*` class in order to keep consistency. It's also closer to how Bootstrap generates these classes and replicating that structure allows us to modify the class with more ease !